tests/e2e.test.js

```diff
@@ -417,13 +417,22 @@ describe('Grid item', () => {
     const clipboardValue = await getClipboardValue(page);
     expect(clipboardValue).toMatch(SVG_REGEX);
   });
+  /*
+  it("is possible to download an icon in all existing file types", async () => {
+    const fileTypes = ["download-type-svg", "download-type-pdf"];
 
-  it('is possible to download an icon as SVG', async () => {
-    await expect(page).toClick('a[download][href$="svg"]');
+    fileTypes.forEach(async (fileType) => {
+      await expect(page).toClick(`button#${fileType}`);
+      await expect(page).toClick(`a[download].download-type-file`);
+    });
   });
-
-  it('is possible to download an icon as PDF', async () => {
-    await expect(page).toClick('a[download][href$="pdf"]');
+*/
+  it.each([
+    ['download-type-svg', 'svg'],
+    ['download-type-pdf', 'pdf'],
+  ])('is possible to download an icon in %s file type', async (fileType) => {
```

Comment:
I don't think this works as you want it to work, you'd need to do:

```suggestion
  it.each([
    ['svg', 'download-type-svg'],
    ['pdf', 'download-type-pdf'],
  ])('is possible to download an icon in %s file type', async (_, fileType) => {
```

But, at least to me, something like the following is clear enough:

```suggestion
  it.each([
    'download-type-svg',
    'download-type-pdf',
  ])('is possible to download an icon (%s)', async (fileType) => {
```